tools/update_rust_version.sh

```diff
@@ -29,9 +29,9 @@ echo Updating Rust to $NIGHTLY
 echo $NIGHTLY > rust-toolchain
 
 # Update all relevant files with the new version string.
-sed -i ''  "s/nightly-[0-9]*-[0-9]*-[0-9]*/${NIGHTLY}/g" .travis.yml
-sed -i ''  "s/nightly-[0-9]*-[0-9]*-[0-9]*/${NIGHTLY}/g" .vscode/settings.json
-sed -i ''  "s/nightly-[0-9]*-[0-9]*-[0-9]*/${NIGHTLY}/g" doc/Getting_Started.md
-sed -i ''  "s/nightly-[0-9]*-[0-9]*-[0-9]*/${NIGHTLY}/g" rust-toolchain
-sed -i ''  "s/nightly-[0-9]*-[0-9]*-[0-9]*/${NIGHTLY}/g" tools/netlify-build.sh
-sed -i ''  "s/[0-9]*-[0-9]*-[0-9]*/${BEST_DATE}/g" shell.nix
+sed -i "s/nightly-[0-9]*-[0-9]*-[0-9]*/${NIGHTLY}/g" .travis.yml
```

Comment:
Ugh.

This is Mac / BSD `sed` versus Unix / coretools `sed`. On Mac, this change doesn't work:
```
$ sed -i "s/nightly-[0-9]*-[0-9]*-[0-9]*/${NIGHTLY}/g" .travis.yml
sed: 1: ".travis.yml": invalid command code .
```

I've had to fix this before, let me track down the solution and I'll push an update to this branch